src/popup.html

```diff
@@ -48,12 +48,29 @@ <h1 class="settings-hl i18n-content" data-i18n-message-id="popupSettingsTitle"><
       </sign-up-panel>
 
       <main-panel class="signed-in-panel hidden">
-        <p class="fx-relay-font-met aliases-remaining remaining-plural"></p>
+        <span class="panel-status hidden">
+          <p class="aliases-remaining remaining-plural"></p>
+          <a 
+            class="premium-cta i18n-content close-popup-after-click button get-premium-link" 
+            data-event-action="click"
+            data-i18n-message-id="popupGetUnlimitedAliases"
+            href=""
```

Comment:
Is it possible to have a fallback URL? In case we can't set the HREF with the correct params, we might default to taking them to the main website: `relay.firefox.com`

This could even bleed over into a special page from relay for a ` /add-on-upgrade` or something in the future. 

cc/ @groovecoder